components/a/test/test_a.cc

```diff
@@ -45,8 +35,9 @@ TEST_P(CompATest, FunctionA)
     /* Arrange */
     ComponentAParameters param = GetParam();
 
-    FUNC_GETB1_RETURN_VALUE = param.funcGetB1ReturnValue;
-    FUNC_GETB2_RETURN_VALUE = param.funcGetB2ReturnValue;
+    CREATE_MOCK(myMock);
+    ON_CALL(myMock, getB1()).WillByDefault(Return(param.funcGetB1ReturnValue));
+    ON_CALL(myMock, getB2()).WillByDefault(Return(param.funcGetB2ReturnValue));
```

Comment:
I was thinking to leave the old code in this file. After SPL is installed and the tests are built, there would be an error that `getB1()` and `getB2()` are defined multiple times. After that, they can fix the test and we can continue with more complicated example. This way we would need to present them the solution immediately, but it can stay.